.circleci/config.yml

```diff
@@ -11,7 +12,21 @@ jobs:
       - node/install-packages
       - run: npm install
       - run: npm test
+  python-build-and-test:
+    executor: python/default
+    steps:
+      - checkout
+      - python/install-deps
+      - run: pip install flake8
+      - run: pip install black
+      - run:
+          name: Check format with Black
+          command: black .
```

Comment:
I'm pretty sure this just reformats stuff, which isn't what we want (we want it to emit an error in the case that formatting failed)

You can call black automatically via flake8-black, which would make this less important (this is what mozregression does: https://github.com/mozilla/mozregression). See:

https://pypi.org/project/flake8-black/